types/react-tracking/test/react-tracking-with-types-tests.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { Track, track as _track, TrackingProp } from 'react-tracking';
+import { Track, track as _track, TrackingProp, Options, Decorator } from 'react-tracking';
```

Comment:
This doesn’t really seem necessary when they are not being used, but adding the exports doesn’t really require additional test coverage imo.